### PR TITLE
K8SPXC-1243 fix deletion from server inventory on pod termination

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-1243](https://jira.percona.com/browse/K8SPXC-1243) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

*  use 'pmm-admin unregister' command to the instance from server inventory on pod termination
*  test case was added

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [x] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Are the manifests (crd/bundle) regenerated if needed?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PXC version?
- [x] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/demand-backup-8-0.log) |
| haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/haproxy-5-7.log) |
| haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/haproxy-8-0.log) |
| ignore-labels-annotations-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/ignore-labels-annotations-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/init-deploy-8-0.log) |
| limits-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/ea735607/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/ea73560784d95150d9494b7028fd3797d7f82a40
image: `perconalab/percona-xtradb-cluster-operator:PR-1416-ea735607`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-8-0.log) |
| haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/haproxy-5-7.log) |
| haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/haproxy-8-0.log) |
| ignore-labels-annotations-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/ignore-labels-annotations-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/50d4c76f801dad53a8f6ebe6816bdb503f2fc89a
image: `perconalab/percona-xtradb-cluster-operator:PR-1416-50d4c76f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/haproxy-8-0.log) |
| ignore-labels-annotations-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/ignore-labels-annotations-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/50d4c76f/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/50d4c76f801dad53a8f6ebe6816bdb503f2fc89a
image: `perconalab/percona-xtradb-cluster-operator:PR-1416-50d4c76f`

File: e2e-tests/monitoring-2-0/run [hunk 1]
```diff
@@ -7,6 +7,25 @@ test_dir=$(realpath $(dirname $0))
...
+get_node_id_from_pmm() {
+	local -a nodeList=()
+	for instance in $(kubectl_bin get pods --no-headers -l app.kubernetes.io/component=pxc --output=custom-columns='NAME:.metadata.name'); do
+		nodeList+=($(kubectl_bin exec -n "$namespace" $instance -c pmm-client -- pmm-admin status --json | jq -r '.pmm_agent_status.node_id'))
+	done
+
+	echo "${nodeList[@]}"
+}
+
+is_node_id_exists() {
```
Comment (tplavcic) [inline e2e-tests/monitoring-2-0/run:19]: maybe `check_node_id_exists` would be better or even `does_node_id_exists`?

File: e2e-tests/monitoring-2-0/run [hunk 2]
```diff
@@ -87,6 +106,27 @@ get_metric_values haproxy_backend_active_servers $namespace-$cluster-haproxy-0 a
...
+	if [ -z "$node_id" ]; then
+		echo "Can't get $node_id node_id from PMM server"
+		exit 1
+	fi
+done
+
+kubectl_bin patch pxc ${cluster} --type json -p='[{"op":"add","path":"/spec/pause","value":true}]'
+wait_for_delete "pod/${cluster}-pxc-0"
+
+is_nodes_id_exist_in_pmm=($(is_node_id_exists "${nodeList[@]}"))
```
Comment (tplavcic) [inline e2e-tests/monitoring-2-0/run:122]: `node_id_exists_pmm` or if we want a longer version `does_node_id_exists_in_pmm`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: e2e-tests/monitoring-2-0/run (hunk 2 above)
Comment (hors) [reply e2e-tests/monitoring-2-0/run]: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/b19ff569e811763cde0a5ba8a2c67fe6dac8ffde done

File: e2e-tests/monitoring-2-0/run (hunk 1 above)
Comment (hors) [reply e2e-tests/monitoring-2-0/run]: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/b19ff569e811763cde0a5ba8a2c67fe6dac8ffde done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/auto-tuning-8-0.log) |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/haproxy-8-0.log) |
| ignore-labels-annotations-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/ignore-labels-annotations-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/limits-8-0.log) |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/pitr-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1416/c0657da9/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/c0657da9e2312c1ebfe9441e149f5b6836a0b1a9
image: `perconalab/percona-xtradb-cluster-operator:PR-1416-c0657da9`

**Review by inelpandzic — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**